action.yml

```
@@ -59,11 +59,33 @@ runs:
         fi
     - shell: bash
       run: pip install -r requirements_dev.txt
+    - shell: bash
+      run: |
+        if [ -f "requirements_ucc.txt" ]
```

You are missing closing brackets for the if statement
```suggestion
        if [ -f "requirements_ucc.txt" ] ;
```